meson.build

```diff
@@ -66,6 +66,8 @@ gtk_layer_shell = dependency('gtk-layer-shell-0',
         required: get_option('gtk-layer-shell'),
         fallback : ['gtk-layer-shell', 'gtk_layer_shell_dep'])
 systemd = dependency('systemd', required: get_option('systemd'))
+date_dep = dependency('date', default_options : [ 'use_system_tzdb=true' ], fallback: [ 'date', 'date_dep' ])
+tz_dep = dependency('date', default_options : [ 'use_system_tzdb=true' ], fallback: [ 'date', 'tz_dep' ])
```

Comment:
Remove this one :)

Reply:
It looks like `date_dep` isn't required (I'll remove it) but `tz_dep` is (otherwise we fail to link with missing `date::locate_zone`).